ADAL/tests/app/src/mac/ADTestAppAcquireTokenWindowController.h

```diff
@@ -29,7 +29,7 @@
 @interface ADTestAppAcquireTokenWindowController : NSWindowController
 {
     IBOutlet NSView* _authView;
-    IBOutlet WebView* _webView;
+    IBOutlet WKWebView* _webView;
```

Comment:
WKWebView is not supported on older OS-s as IBOutlet (in storyboard). I think you might still be using WebView in storyboard or how can it work?

Reply:
fixed